mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
```

Comment:
Also add mypy to Python 3 development dependencies.

For instance, `requirements/dev-3.txt`.

Reply:
Done